collections/_article/rendering-priorities-september-2024.md

```diff
@@ -0,0 +1,115 @@
+
+* **RenderingDeviceDriver refactor ([GH-83452](https://github.com/godotengine/godot/pull/83452)) (Done)**
+
+This was very important foundational work that has allowed us to simplify our rendering code and ease the future maintenance burden as we port to more rendering backends.
+
+* **Acyclic render graph optimization ([GH-84976](https://github.com/godotengine/godot/pull/84976)) (Done)**
+
+As expected we merged the new Acyclic Render graph in 4.3. It significantly improved stability of the engine and resolved many hard to fix errors as well as moderately improved performance. To read more see the [blog post](https://godotengine.org/article/rendering-acyclic-graph/).
+
+* **Compositor Effects (Rendering Hooks) ([GH-80214](https://github.com/godotengine/godot/pull/80214)) (Done)**
+
+The new CompositorEffects API allows you to register callbacks that will be called in the middle of the rendering process. This lets you insert custom RenderingDevice commands in the middle of the render frame to customize rendering.
+
+Right now the API is functional and exposes almost all internal render state so you can access all the intermediate information.
+
+There is already an [official demo](https://github.com/godotengine/godot-demo-projects/tree/master/compute/post_shader) so you can see how it works!
+
+* **Metal rendering driver [GH-88199](https://github.com/godotengine/godot/pull/88199)(Merged for 4.4)**
+
+This was a work in progress for much of the 4.3 release cycle and was finished slightly too late to include in 4.3. We merged it immediately for 4.4 so we can get widespread testing before release. Its ready to try in [4.4-dev1](https://godotengine.org/article/dev-snapshot-godot-4-4-dev-1/) or later releases.
+
+Metal enhances both performance and stability on Apple silicon devices. Please test it out and let us know if you run into any issues!
+
+* **SDFGI refactor ([GH-86267](https://github.com/godotengine/godot/pull/86267)) (In progress)**
+
+This is still a work in progress, but results are already quite good. Both performance and quality are better than SDFGI. Juan still has many ideas he wants to apply, so this work may continue for some time before it is ready.
+
+## New Priorities
+
+We have completed almost all of our previous priorities, so now we get to set out new goals for the next few release cycles. Right now rendering contributors are most excited about two things:
+
+* Unblocking advanced users (especially for advanced VFX), and
+
+* Improving performance.
+
+These are rising to the top of the priority list as Godot becomes increasingly stable and the core user-experience feels quite good.
+
+In particular the following are areas where you can expect to see changes in the next couple of releases. Bear in mind that we don't have dedicated people working on any of these priorities. If you are interested in contributing, please consider contributing in one of these areas.
+
+* **Performance:**
```

Comment:
Other items don't have a colon at the end, so I suggest removing it here (or adding it to the other items below):

```suggestion
* **Performance**
```